static/usr/lib/systemd/system/snapd.autoimport-device@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Auto import assertions from a specific block device
+After=snapd.service snapd.socket snapd.seeded.service
+# don't run on classic
+ConditionKernelCommandLine=|snap_core
+# TODO:UC20: only enable this in run mode?
+ConditionKernelCommandLine=|snapd_recovery_mode
```

Comment:
@anonymouse64 wdyt about this?

Reply:
I will set it to `snapd_recovery_mode=run`. Recovery and install should not do anything interesting. Unless someone wants to login during the install.

Reply:
Changed

Reply:
Right, I'm +1 on this. I think that you can get a shell in recovery without this iirc?